vowpalwabbit/conditional_contextual_bandit.cc

```diff
@@ -529,23 +596,36 @@ void finish(ccb& data)
 {
   data.actions.~vector<example*>();
   data.slots.~vector<example*>();
-  data.origin_index.~map<uint32_t,uint32_t>();
+  data.origin_index.~map<uint32_t, uint32_t>();
```

Comment:
What is this map for?  It seems best to avoid map if we can.

Reply:
Origin index is used to track the original index of each action as they are removed after each iteration. Some sort of book keeping of this type is required.